_episodes/11-hpc-intro.md

```diff
@@ -40,83 +34,61 @@ they started:
-a CLI is a **read-evaluate-print loop**, or REPL: when the user types a command and then presses the
-Enter (or Return) key, the computer reads it, executes it, and prints its output. The user then
-types another command, and so on until the user logs off.
-
-Learning to use Bash or any other shell sometimes feels more like programming than like using a
-mouse. Commands are terse (often only a couple of characters long), their names are frequently
-cryptic, and their output is lines of text rather than something visual like a graph. However, using
-a command line interface can be extremely powerful, and learning how to use one will allow you to
-reap the benefits described above.
-
-## The rest of this lesson
-
-The only way to use these types of resources is by learning to use the command line. This
-introduction to HPC systems has two parts:
-
-* We will learn to use the UNIX command line (also known as Bash).
-* We will use our new Bash skills to connect to and operate a high-performance computing
-  supercomputer.
-
-The skills we learn here have other uses beyond just HPC - Bash and UNIX skills are used everywhere,
-be it for web development, running software, or operating servers. It's become so essential that
-Microsoft now ships as [part of Windows][_ms-ubuntu])! Knowing how to use Bash and HPC systems will
-allow you to operate virtually any modern device. With all of this in mind, let's connect to a
-cluster and get started!
-
-[_ms-ubuntu]: https://www.microsoft.com/en-us/store/p/ubuntu/9nblggh4msv6
+
+> # And what do you do?
+> 
+> Talk to your neighbour, office mate or [rubber duck](https://rubberduckdebugging.com/) about your research. How does computing help you do your research? 
+> How could more computing help you do more or better research?
+{: .callout }
+
+
+## Doing Analysis or Running Code
+
+### A standard Laptop for standard tasks
+
+Today, people coding or analysing data typically work with laptops.
+ 
+{% include figure.html url="" max-width="20%" file="/fig/200px-laptop-openclipartorg-aoguerrero.svg" alt="A standard laptop"
+caption="" %}
+
+While the following discussion may appear silly at first glance, let's dissect what resources programs running on a laptop require:
+- the keyboard and touchpad is used to provide flags and options
+- the display is shows progress and results
+- the laptops internal computing power and memory is used to perform calculations
+- offen GUI style human interfaces are employed
+
+### When tasks take too long
+
+When the task to solve become more heavy on computations, the operations are typically out-sourced from the local hardware in the laptop. Take for example the task to find the directions for your next business trip. The capabilities of your laptop are typically not enough to calculate that route spontaneously. So you use website, which in turn runs on a server that is almost exclusively no in the same room than you are.
```

Comment:
hardware = jargon that needs to be defined.  What do you mean?

Reply:
removed that bit to avoid confusion.